src/pages/Home/index.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import axios from "axios";
+import LazyLoad from "react-lazyload";
+import Loading from "../../components/Loading";
+import { Container, StyledLink } from "./home.styles";
+
+class Home extends React.Component {
+  state = {
+    data: null,
+    isLoading: false,
+    pageToLoad: 1,
+  };
+
+  retrievePhotos = async () => {
+    try {
+      this.setState({ isLoading: true });
+      const { data } = await axios(
+        `https://api.unsplash.com/photos?page=${this.state.pageToLoad}&per_page=50&client_id=${process.env.REACT_APP_API_KEY}`
+      );
+      this.setState({ data: data, isLoading: false });
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  componentDidMount() {
+    this.retrievePhotos();
+  }
+
+  render() {
+    return (
+      <>
+        {this.state.isLoading && <Loading />}
+        <Container>
+          {this.state.data &&
+            !this.state.isLoading &&
+            Object.entries(this.state.data).map((entry) => {
```

Comment:
You can use Objec.values here :D

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values